news/2023/2023-09-09-monthly-beatmapping-contest-september-2023.md

```diff
@@ -0,0 +1,118 @@
+
+Your biggest struggle might be finding a song packed with enough variation to make a cool beatmap. Luckily, there's *a lot* of songs to choose from:
+
+### Songs
+
+#### osu!
+
+Any song listed on [osu!'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) is eligible for this contest!
+
+Feel free to cut longer songs to a length that's easier to map. [Talk to pishifat](https://osu.ppy.sh/community/chat?sendto=3178418) if you need help with that!
+
+#### osu!mania
+
+Any [Featured Artist](https://osu.ppy.sh/beatmaps/artists) song that is at least 4 minutes is eligible for this contest! [This search criteria](https://osu.ppy.sh/beatmaps/artists/tracks?is_default_sort=true&sort=update_desc&length%5Bgte%5D=240) includes every song that's 4 minutes or longer, so your song must be in this list!
+
+## Rewards
+
+The Monthly Beatmapping Contest rewards users with [contest points](/wiki/Contests/Contest_points) based on their placements:
+
+- **Judging winner:** 2 points
+- **Judging runner-up:** 1 point
+- **Screening crew's #1 pick:** 1 point
+
+These numbers may vary depending on how many submissions the contest receives. Check [the wiki article](/wiki/Contests/Contest_points) for details.
+
+Contest points reward users at certain thresholds:
+
+- **Profile badges** are attainable at 2, 4, and 6 points each. The badges are tiered, meaning each new badge will replace the previous one!
+
+![](/wiki/shared/news/2020-02-20-monthly-beatmapping-contest-1-results/mbc-badge-1.png)
+
+- ***Elite Mapper* forum title** is attainable at 6 points.
+
+![](/wiki/shared/news/2023-09-04-monthly-beatmapping-contest-september-2023/elitemapper.png)
+
+Additionally, a **Mappers' Guild keychain** is optionally rewarded to the judging winner of each month's contest. Its design is based on your beatmap!
+
+![keychain](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/keychain.jpg)
+
+If you're interested in receiving these keychains outside of a contest situation, check out [these recent changes to the Mappers' Guild](https://osu.ppy.sh/home/news/2023-08-29-changes-to-the-mappers-guild)!
+
+## Rules
+
+Ready to map? Good. Here's what you need to keep in mind:
+
+- **Submissions must follow the month's beatmapping [limitation](#limitation):**
+  - **osu!:** every segment of a slider must be curved.
+  - **osu!mania:** song must be at least 4 minutes.
+- **Submissions must be a single difficulty from one of the month's [selected songs](#songs).**
+- **Submissions must be in `.osz` format.**
+- **Submissions must NOT be uploaded to the osu! website until after the contest's results are finalized.**
+- **Submissions must abide by the [ranking criteria](/wiki/Ranking_criteria).**
+
+Note that all submissions will be anonymized before they're reviewed by screeners/judges to avoid bias. During this process, storyboards and backgrounds may be removed.
```

Comment:
```suggestion
Note that all submissions will be anonymised before they're reviewed by screeners/judges to avoid bias. During this process, storyboards and backgrounds may be removed.
```